packages/hub/test/hub.test.ts

```diff
@@ -270,14 +262,12 @@ describe('Hub', () => {
       expect(args[0]).toBe('a');
     });
 
-    test('should set event_id in hint', () => {
+    test('should get event_id from client', () => {
       const testClient = makeClient();
       const hub = new Hub(testClient);
 
-      hub.captureMessage('a');
-      const args = getPassedArgs(testClient.captureMessage);
-
-      expect(args[2].event_id).toBeTruthy();
+      const id = hub.captureMessage('a');
+      expect(id).toBeTruthy();
```

Comment:
l: Probably related to my point above but can we check the id here for a more precise value than it being truthy or is this alright (totally your call, just asking)?